common/shlibs

```diff
@@ -3875,3 +3875,4 @@ libhidrd_opt.so.0 hidrd-0.2.0_1
 libhidrd_strm.so.0 hidrd-0.2.0_1
 libhidrd_fmt.so.0 hidrd-0.2.0_1
 libjcat.so.1 libjcat-0.1.2_1
+libthrift-0.13.0.so thrift-devel-0.13.0_4
```

Comment:
No, we don't want to pull `*-devel` for normal dependency.

Reply:
done